src/organisms/gv-newsletter-subscription.js

```diff
@@ -0,0 +1,276 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { css, html, LitElement } from 'lit-element';
+import { dispatchCustomEvent } from '../lib/events';
+
+/**
+ * The newsletter subscription page
+ *
+ * @fires gv-newsletter-subscription:subscribe - Custom event when link click
+ * @fires gv-newsletter-subscription:skip - Custom event when link click
+ *
+ * @attr {String} title - The title of newsletter subscription container
+ * @attr {String} header - The header of newsletter subscription container
+ * @attr {String} emailPlaceholder - The placeholder used for the email input
+ * @attr {String} subscribeLabel - The text of the subscribe button
+ * @attr {String} skipLabel - The text of the skip button
+ * @attr {Array} taglines - The taglines to promote the newsletter subscription
+ * @attr {String} email - The email value
+ * @attr {Boolean} emailDisabled - The disabled state of email input
+ * @attr {String} githubUrl - The url to github page
+ * @attr {String} twitterUrl - The url to twitter page
+ * @attr {String} linkedinUrl - The url to linkedin page
+ *
+ */
+export class GvNewsletterSubscription extends LitElement {
+
+  static get properties () {
+    return {
+      title: { type: String },
+      header: { type: String },
+      emailPlaceholder: { type: String, attribute: 'email-placeholder' },
+      subscribeLabel: { type: String, attribute: 'subscribe-label' },
+      skipLabel: { type: String, attribute: 'skip-label' },
+      taglines: { type: Array },
+      email: { type: String },
+      emailDisabled: { type: Boolean },
+      githubUrl: { type: String },
+      twitterUrl: { type: String },
+      linkedinUrl: { type: String },
+    };
+  }
+
+  static get styles () {
+    return [
+      // language=css
+      css`
+        .newsletter-container {
```

Comment:
On each component we initialize two default properties

`:host {
  box-sizing: border-box;
  margin: 0.2rem;
}`